k8s/postgresql/README.md

```diff
@@ -208,6 +223,46 @@ PGPASSWORD=$(kubectl get secret "${APP_INSTANCE_NAME}-secret" --output=jsonpath=
 echo PGPASSWORD=$PGPASSWORD sslmode=require psql -U postgres -h $EXTERNAL_IP
 ```
 
+# Application metrics
+
+## Prometheus metrics
+
+The application is configured to expose its metrics through
+[PostgreSQL Prometheus.io exporter plugin](https://github.com/wrouesnel/postgres_exporter)
+in the [Prometheus format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md).
+Metrics can be read on a single HTTP endpoint available at `[POSTGRESQL_CLUSTER_IP]:9187/metrics`,
+where `[POSTGRESQL_CLUSTER_IP]` is the IP address of the application on Kubernetes cluster.
+
```

Comment:
It would be nice to have here an information about the impact of password change.

Reply:
It will change nothing.